app/assets/stylesheets/styles.scss

```diff
@@ -0,0 +1,31 @@
+h1,
+h2,
+h3 {
+  font-weight: lighter;
+}
+
+h1 a {
+  color: #222222;
```

Comment:
Color literals like `#222222` should only be used in variable declarations; they should be referred to via variable everywhere else.